client/style.css

```diff
@@ -0,0 +1,14 @@
+.kanban-board {
```

Comment:
This particular css file is generated from scss by gulp, but ok, I will include it in gitignore file.